js/main.js

```diff
@@ -1,6 +1,36 @@
 'use strict';
 
 // anime.js利用
+/* ハンバーガーメニュー */
+const menuOpen = document.getElementById('open');
+const menuClose = document.getElementById('close');
+const overlayMenu = document.querySelector('.header__overlay');
+const backgroundMask = document.getElementById('blackBg')
+
+menuOpen.addEventListener('click', function (e) {
+  openOverlayMenu();
+});
```

Comment:
```suggestion
menuOpen.addEventListener('click', function () {
  openOverlayMenu();
});
```

この引数は必要ないですよね…？（違ったらすいません）

Reply:
MDN　イベントの紹介
https://developer.mozilla.org/ja/docs/Learn/JavaScript/Building_blocks/Events

上記を参考に(e)としていたんですが、考えたら関数内で引数を使うわけではないので（e.target.style～とかで指定する事がない）不要でしたね・・・！ありがとうございます。